Gemfile.lock

```diff
@@ -25,10 +25,12 @@ GEM
     ethon (0.12.0)
       ffi (>= 1.3.0)
     eventmachine (1.2.7)
+    eventmachine (1.2.7-x64-mingw32)
```

Comment:
should these changes be checked in?

Reply:
They're automatically added when run on a windows machine. They don't seem to cause an issue when generating the pages on github so I left them in there.